Spreadsheets (#1783)
enkidevs/curriculum: excel/beginner/charts/types-of-charts-ii.md

---
author: Stefan-Stojanovic

levels:
  - beginner

aspects:
  - new
  - workout

type: normal

category: how to

---

# Types of charts continued

---
## Content

> Some other famous charts are the `line` and `combo` charts.

Reviewer: ```suggestion
Some other well known charts are the `line` and `combo` charts.
```

> We will the same data from the previous insight.

Reviewer: ```suggestion
We will use the same data from the previous insight.
```

Data used:
![chart-data](https://img.enkipro.com/5fb6d6cec47fd3c56f26b1e8349d8a17.png)

### The `Line` Chart

> The Line chart is made with lines and plot points. These points create clear visualization of the increase and decrease of, for instance, players. 

Reviewer: ```suggestion
The Line chart is made with lines and plot points. These points create clear visualization of the increase and decrease of, for instance, players in December. 

```

![line](https://img.enkipro.com/f0db10f741888c074552309a2c133e10.png)

### The `Combo` Chart

A combination chart is a chart that combines two or more chart types in a single chart.

Using the same chart data from above, we are going to add a revenue column:

![with-revenue-plus](https://img.enkipro.com/522f9e74fe05fb92235acf71963544e4.png)

> And create a combo 'Line-column` chart.

Reviewer: ```suggestion
And create a combo `Line-column` chart.

```
